helm/kiam/values.yaml

```diff
@@ -133,6 +133,23 @@ agent:
     #   hostPath: /usr/share/ca-certificates
     #   readOnly: true
 
+  ## Agent initContainers
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+  ##
+  initContainers: []
+    ## Check kiam-server dns hostname
+    # - name: check-dns
+    #   image: busybox:1.31
+    #   imagePullPolicy: IfNotPresent
+    #   command: ['sh', '-c', 'until nslookup kiam-server; do sleep 1; done;']
+    #   resources:
+    #     requests:
+    #       cpu: 20m
+    #       memory: 16Mi
+    #     limits:
+    #       cpu: 20m
+    #       memory: 16Mi
+
```

Comment:
I think having the option to configure init containers if you desire is great, but I wouldn't want this example included in the repo, it's very specific to a problem you were experiencing in your cluster

Reply:
No problem, i removed!